github/repos_test.go

```diff
@@ -42,15 +42,17 @@ func TestRepositoriesService_List_specifiedUser(t *testing.T) {
 		testMethod(t, r, "GET")
 		testHeader(t, r, "Accept", mediaTypeLicensesPreview)
 		testFormValues(t, r, values{
-			"type":      "owner",
-			"sort":      "created",
-			"direction": "asc",
-			"page":      "2",
+			"visibility":  "public",
+			"affiliation": "owner,collaborator",
+			"type":        "owner",
```

Comment:
Done. Broke this out into 2 different tests so that both RepositoryListOptions are valid.